src/System.Private.ServiceModel/tools/test/Bridge/Bridge/Program.cs

```diff
@@ -21,6 +21,8 @@ internal class Program
 
         private static void Main(string[] args)
         {
+            AppDomain.CurrentDomain.UnhandledException +=
+                new UnhandledExceptionEventHandler(BridgeUnhandledExceptionHandler);
```

Comment:
@mconnew is correct above. However, the exception will be passed back to here only if the exception is serializable (has SerializableAttribute).